readme.md

```
@@ -230,67 +113,20 @@ Common error types:
+**See The [scrapers folder](app/scrapers)**
 
 ## 📄 License
 
-This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
+This project is licensed under the The UnlicenseLicense - see the [LICENSE](LICENSE) file for details.
 
-## 👥 Team
+## Resource
 
-**NB Team** - A community-driven development team focused on creating useful tools and libraries.
+**NB Scraper is desaign to work for easy to use**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)